### ISSUE TITLE
Mask sequences with trailing `N`

### ISSUE THREAD

**Issue by ohickl** [althonos/pyrodigal#55]
Hi,
thanks for the great tool!

I noticed that pyrodigal would still output translations of sequences with trailing `N`, despite setting `mask=True`. Is this intended?
Setup:
```
...
gene_finder = pyrodigal.GeneFinder(meta=True, mask=True, min_mask=0)
...
for id, seq in zip(headers, sequences):
    genes = gene_finder.find_genes(seq)
    predictions.append((id, genes))
...
with gzip.open(output_file, "wt") as f:
    for contig_id, genes in predictions:
        genes.write_translations(f, sequence_id=contig_id, include_stop=False)
...
```
Tested with e.g. [https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_902164675.1](https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_902164675.1).

Or am I maybe running it wrong?

Best

Oskar

**Comment by althonos** [comment]
Hi Oskar, this was totally a bug inside the masking code, I will push a patch.

**Comment by althonos** [comment]
Fixed in `v3.4.0` 👍

**Comment by ohickl** [comment]
Great, thanks!
